Local test failure: front end routes - #https get http://127.0.0.1:3030/legal/terms

× node - front end routes - #https get http://127.0.0.1:3030/legal/terms
AssertionError: #https get http://127.0.0.1:3030/legal/terms: https://haveibeenpwned.com/: expected 403 to equal 200
  at <tests/server/helpers/routesHelpers.js:174:16>

The response to haveibeenpwned.com is 403, I suspect we are being blocked because we aren't sending any headers that indicates we are a real browser.

--- a/packages/fxa-content-server/tests/server/helpers/routesHelpers.js
+++ b/packages/fxa-content-server/tests/server/helpers/routesHelpers.js
@@ -140,6 +140,8 @@ const IGNORE_URL_REGEXPS = [
   /support\.mozilla\.org/,
   // skip the livereload link in the mocha tests
   /localhost:35729/,
+  // haveibeenpwned 403s.
+  /haveibeenpwned\.com/,
 ];
 
 function isUrlIgnored(url) {

